shared/util/errors.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+class RPCError extends Error {
```

Comment:
yeah instance of failing.. I'll put a comment in errors.js about this and that a fix will be going in soon.